vaadinfrontend/src/main/java/life/qbic/datamanager/views/project/view/components/ProjectDetailsComponent.java

```diff
@@ -140,20 +146,27 @@ public Handler(ProjectDetailsComponent component,
 
     private void contactClickListener() {
       //when clicking the layout switch!
-      component.projectManagerLayout.addClickListener(click ->{
-        component.pmElement.setVisible(false);
```

Comment:
why dont we set the pm element as invisible here? Logically its one mechanism, right?

Reply:
Why is the focus listener the better option to choose?

Reply:
the focus listener is the better choice as we want to have a blur listener (as soon as the user does not edit the field anymore or focusses on another element, the display shall change)
Focus and blur are opposites.

Reply:
I thought on it again and you are right, we do not necessarily need the focus listener as long as we call the focus method.